library_samples/Python3/ocs_sample_library_preview/DataViews.py

```diff
@@ -156,92 +157,110 @@ def getDataViews(self, namespace_id, skip=0, count=100):
             results.append(DataView.fromJson(t))
         return results
 
-    def getDataGroups(
-        self, namespace_id, dataView_id, skip=0, count=100, returnAsDynamicObject=False
+    def getResolvedDataItems(
+        self, namespace_id, dataView_id, query_id
     ):
         """
         Retrieves all of the DataGroups from the specified DataView from
```

Comment:
Documentation no longer correct, this should say it retrieves Data Items not Data Groups